### PR TITLE
Timeout all the things

### PR DESCRIPTION
**Pull request by Licenser**
# Pull request

## Description

Introduces 5-minute timeout for all serial tests

## Checklist

* [x] The RFC, if required, has been submitted and approved
* [x] Any user-facing impact of the changes is reflected in docs.tremor.rs
* [x] The code is tested
* [x] Use of unsafe code is reasoned about in a comment
* [x] Update CHANGELOG.md appropriately, recording any changes, bug fixes, or other observable changes in behavior
* [x] The performance impact of the change is measured (see below)

## Performance

Three cakes

### CODE REVIEW

File: tremor-pipeline/src/metrics.rs [hunk 1]
```diff
@@ -74,3 +74,25 @@ pub fn value(
         TIMESTAMP: timestamp
     })
 }
+
+#[cfg(test)]
+mod test {
```
Comment (mfelsche) [inline tremor-pipeline/src/metrics.rs:79]: how is this related?

**Review by mfelsche — APPROVED** [review]

File: tremor-pipeline/src/metrics.rs (hunk 1 above)
Comment (Licenser) [reply tremor-pipeline/src/metrics.rs:79]: It's not, I don't know how this got here :joy:

**Review by mfelsche — APPROVED** [review]
